workflow_array_ephys/ingest.py

```diff
@@ -71,18 +77,22 @@ def ingest_sessions(session_csv_path='./user_data/sessions.csv'):
         session_key = {'subject': sess['subject'], 'session_datetime': min(session_datetimes)}
         if session_key not in session.Session():
             session_list.append(session_key)
-            session_dir_list.append({**session_key, 'session_dir': session_dir.relative_to(root_data_dir).as_posix()})
+            root_dir = element_data_loader.utils.find_root_directory(
+                                                    get_ephys_root_data_dir(), sess_dir)
+            sess_dir_list.append({**session_key, 'session_dir': sess_dir.relative_to(root_dir).as_posix()})
             probe_insertion_list.extend([{**session_key, **insertion} for insertion in insertions])
 
     print(f'\n---- Insert {len(session_list)} entry(s) into session.Session ----')
-    session.Session.insert(session_list)
-    session.SessionDirectory.insert(session_dir_list)
+    # Broz 102821 - prev, skip_dupes was true for ingest_subj, but not ingest_sess
+    #               I thought they should mirror each other, chose both True
```

Comment:
```suggestion
```
These comments are great to understand your thought process.  Please remove from the code and add it to the `Conversation` tab in the pull request.  Thanks.

Reply:
For `session.Session` and `session.SessionDirectory`, I don't believe `skip_duplicates=True` is required because of the following line:
```
if session_key not in session.Session():
```